PI-Vision-Extensibility/README.md

```diff
@@ -0,0 +1,21 @@
+# PI Vision Extensibility
+This repository contains sample HTML/CSS/JavaScript for creating custom symbols for PI Vision.
+
+## Overview
+The sample code contained in this repository provides code to extend PI Vision with new custom symbols.  
```

Comment:
I think this line is off and I would say something like inside this repository are custom symbol samples and the PI Vision Extensiblity guide, which is available under <a href="docs">.docs</a>

Reply:
OK, will make this change